### DISCUSSION TITLE
Nolinting specific go vet check

### DISCUSSION THREAD

**Discussion by anuramat** [golangci/golangci-lint#3971] [Q&A] [answered]
I want to ignore `fieldalignment` analyzer for some structs, is that possible without `//nolint:govet`?

**Comment by kehiy** [comment, marked as answer]
hello
first of all, using `fieldalignment` is not make sense at all in Golang!
but you have 2 ways to make it done, the first one is to make the file ignored (for `fieldalignment`) and the second one is to use `//nolint:govet`

but IMHO it's better to don't use `fieldalignment` at all!

**Reply by anuramat** [reply]
Thanks!
